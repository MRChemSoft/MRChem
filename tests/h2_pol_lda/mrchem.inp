# vim:syntax=sh:

rel_prec = 1.0e-3                 # Overall relative precision

MPI {
  numerically_exact = true
  share_coulomb_density = true
  share_coulomb_potential = true
  share_nuclear_potential = true
  share_xc_density = false
  share_xc_potential = false
}

MRA {
  min_scale = -5                  # Root scale defines world size 2^{-n}
  corner = [ -1, -1, -1 ]         # Translation of corner root box [x,y,z]
  boxes = [ 2, 2, 2 ]             # Number of root boxes [x,y,z]
}

Molecule {
  angstrom = true
$coords
H      0.0000 0.0000   -0.3705
H      0.0000 0.0000    0.3705
$end
}

WaveFunction {
  method = DFT                    # Wave function method (HF or DFT)
  restricted = false              # Unrestricted calculation
}

DFT {
$functionals
LDA
$end
}

Properties {
  scf_energy = true               # Compute ground state energy
  polarizability = true           # Compute electric polarizability
}

Polarizability {
  frequency = [0.0]               # Static polarizability 
}

SCF {
  run = false                     # Use initial guess, don't optimize
  initial_guess = GTO             # Reading precomputed GTO MOs
}

Response {
  kain = 3                        # Length of KAIN iterative history
  max_iter = 10
  orbital_thrs = 1.0e-2           # Convergence threshold in orbital residual
  directions = [0,0,1]            # Compute only in z direction
}

