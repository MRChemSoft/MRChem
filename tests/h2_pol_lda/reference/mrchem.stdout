                                                                      
                                                                      
**********************************************************************
***                                                                ***
***                                                                ***
***        __  __ ____   ____ _                                    ***
***       |  \/  |  _ \ / ___| |__   ___ _ __ ___                  ***
***       | |\/| | |_) | |   | '_ \ / _ \ '_ ` _ \                 ***
***       | |  | |  _ <| |___| | | |  __/ | | | | |                ***
***       |_|  |_|_| \_\\____|_| |_|\___|_| |_| |_|                ***
***                                                                ***
***       VERSION            0.3.0-alpha                           ***
***                                                                ***
***       Git branch         prop-print                            ***
***       Git commit hash    3bfae167fa1d92f7c42c                  ***
***       Git commit author  Stig Rune Jensen                      ***
***       Git commit date    Mon Mar 16 18:22:17 2020 +0100        ***
***                                                                ***
***       Magnar Bjorgve     <magnar.bjorgve@uit.no>               ***
***       Roberto Di Remigio <roberto.di.remigio@uit.no>           ***
***       Luca Frediani      <luca.frediani@uit.no>                ***
***       Gabriel Gerez      <gsa017@post.uit.no>                  ***
***       Stig Rune Jensen   <stig.r.jensen@uit.no>                ***
***       Peter Wind         <peter.wind@uit.no>                   ***
***                                                                ***
**********************************************************************

----------------------------------------------------------------------

 MPI processes         :      (no bank)                             1
 OpenMP threads        :                                            1
 Total cores           :                                            1
                                                                      
----------------------------------------------------------------------

XCFun DFT library Copyright 2009-2020 Ulf Ekstrom and contributors.
See http://dftlibs.org/xcfun/ for more information.

This is free software; see the source code for copying conditions.
There is ABSOLUTELY NO WARRANTY; not even for MERCHANTABILITY or
FITNESS FOR A PARTICULAR PURPOSE. For details see the documentation.
Scientific users of this library should cite
U. Ekstrom, L. Visscher, R. Bast, A. J. Thorvaldsen and K. Ruud;
J.Chem.Theor.Comp. 2010, DOI: 10.1021/ct100117s

----------------------------------------------------------------------

 MRCPP version         : 1.2.0-alpha2
 Git branch            : prop-print
 Git commit hash       : 64ff947f46ad4042b90b
 Git commit author     : Stig Rune Jensen
 Git commit date       : Mon Mar 16 16:56:57 2020 +0100

 Linear algebra        : EIGEN v3.3.7
 Parallelization       : OpenMP

----------------------------------------------------------------------


                                                                      
======================================================================
                       MultiResolution Analysis
----------------------------------------------------------------------
 polynomial order      : 5
 polynomial type       : Interpolating
----------------------------------------------------------------------
 total boxes           : 8
 boxes                 : [          2           2           2 ]
 unit lengths          : [   32.00000    32.00000    32.00000 ]
 scaling factor        : [    1.00000     1.00000     1.00000 ]
 lower bounds          : [  -32.00000   -32.00000   -32.00000 ]
 upper bounds          : [   32.00000    32.00000    32.00000 ]
 total length          : [   64.00000    64.00000    64.00000 ]
======================================================================


                                                                      
**********************************************************************
***                                                                ***
***                     Initializing Molecule                      ***
***                                                                ***
**********************************************************************
                                                                      
                                                                      
======================================================================
                               Molecule
----------------------------------------------------------------------
 Charge                :                                            0
 Multiplicity          :                                            1
----------------------------------------------------------------------
    N    Atom          :              x              y              z
----------------------------------------------------------------------
    0       H          :       0.000000       0.000000      -0.700143
    1       H          :       0.000000       0.000000       0.700143
----------------------------------------------------------------------
 Center of mass        :       0.000000       0.000000      -0.000000
 Gauge origin          :       0.000000       0.000000       0.000000
======================================================================


                                                                      
**********************************************************************
***                                                                ***
***              Computing Initial Guess Wavefunction              ***
***                                                                ***
**********************************************************************
                                                                      
                                                                      
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 Calculation           : Reading Gaussian-type MOs
 Precision             : 1.00000e-03
 Restricted            : False
 Basis file            : initial_guess/mrchem.bas
 MO alpha file         : initial_guess/mrchem.moa
 MO beta file          : initial_guess/mrchem.mob
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~


======================================================================
                          Molecular Orbitals
----------------------------------------------------------------------
 Alpha electrons       :                                            1
 Beta electrons        :                                            1
 Total electrons       :                                            2
----------------------------------------------------------------------
    n  Occ Spin        :                                         Norm
----------------------------------------------------------------------
    0    1    a        :                           9.999999974574e-01
    1    1    b        :                           9.999999974574e-01
======================================================================


                                                                      
**********************************************************************
***                                                                ***
***              Computing Ground State Wavefunction               ***
***                                                                ***
**********************************************************************
                                                                      
                                                                      
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 Calculation           : Compute initial energy
 Method                : DFT
 Precision             : 1.00000e-03
 Localization          : Off
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~


======================================================================
                           Molecular Energy
----------------------------------------------------------------------
 Kinetic energy        :           (au)                1.073485982368
 E-N energy            :           (au)               -3.566631902809
 Coulomb energy        :           (au)                1.328364198272
 Exchange energy       :           (au)                0.000000000000
 X-C energy            :           (au)               -0.653870519814
 Ext. field (el)       :           (au)                0.000000000000
 Ext. field (nuc)      :           (au)                0.000000000000
----------------------------------------------------------------------
 Electronic energy     :           (au)               -1.818652241983
 Nuclear energy        :           (au)                0.714139337223
----------------------------------------------------------------------
 Total energy          :           (au)           -1.104512904760e+00
                       :     (kcal/mol)           -6.930923063698e+02
                       :       (kJ/mol)           -2.899898211733e+03
                       :           (eV)           -3.005532594413e+01
======================================================================


======================================================================
                           Orbital Energies
----------------------------------------------------------------------
    n  Occ Spin        :                                      Epsilon
----------------------------------------------------------------------
    0    1    a        :           (au)               -0.345283880497
    1    1    b        :           (au)               -0.345283880497
----------------------------------------------------------------------
 Sum occupied          :           (au)               -0.690567760993
======================================================================


                                                                      
**********************************************************************
***                                                                ***
***             Computing Linear Response Wavefunction             ***
***                                                                ***
**********************************************************************
                                                                      
                                                                      
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 Calculation           : Static linear response
 Method                : DFT
 Perturbation          : h_E_dip[z]
 Max iterations        : 10
 KAIN solver           : 3
 Start precision       : 1.00000e-03
 Final precision       : 1.00000e-03
 Helmholtz precision   : 1.00000e-03
 Property threshold    : Off
 Orbital threshold     : 1.00000e-02
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~


======================================================================
 Iter         MO residual      Symmetric property              Update
----------------------------------------------------------------------
    0        1.000000e+00          0.000000000000        0.000000e+00
    1        2.061933e+00         -5.523958969627       -5.523959e+00
    2        4.485058e-01         -7.124916590397       -1.600958e+00
    3        3.140761e-02         -7.211411969279       -8.649538e-02
    4        5.671671e-03         -7.222012691491       -1.060072e-02
----------------------------------------------------------------------
                   SCF converged in 4 iterations!
======================================================================


                                                                      
**********************************************************************
***                                                                ***
***                 Printing Molecular Properties                  ***
***                                                                ***
**********************************************************************
                                                                      
                                                                      
======================================================================
                               Molecule
----------------------------------------------------------------------
 Charge                :                                            0
 Multiplicity          :                                            1
----------------------------------------------------------------------
    N    Atom          :              x              y              z
----------------------------------------------------------------------
    0       H          :       0.000000       0.000000      -0.700143
    1       H          :       0.000000       0.000000       0.700143
----------------------------------------------------------------------
 Center of mass        :       0.000000       0.000000      -0.000000
 Gauge origin          :       0.000000       0.000000       0.000000
======================================================================


======================================================================
                           Orbital Energies
----------------------------------------------------------------------
    n  Occ Spin        :                                      Epsilon
----------------------------------------------------------------------
    0    1    a        :           (au)               -0.345283880497
    1    1    b        :           (au)               -0.345283880497
----------------------------------------------------------------------
 Sum occupied          :           (au)               -0.690567760993
======================================================================


======================================================================
                           Molecular Energy
----------------------------------------------------------------------
 Kinetic energy        :           (au)                1.073485982368
 E-N energy            :           (au)               -3.566631902809
 Coulomb energy        :           (au)                1.328364198272
 Exchange energy       :           (au)                0.000000000000
 X-C energy            :           (au)               -0.653870519814
 Ext. field (el)       :           (au)                0.000000000000
 Ext. field (nuc)      :           (au)                0.000000000000
----------------------------------------------------------------------
 Electronic energy     :           (au)               -1.818652241983
 Nuclear energy        :           (au)                0.714139337223
----------------------------------------------------------------------
 Total energy          :           (au)           -1.104512904760e+00
                       :     (kcal/mol)           -6.930923063698e+02
                       :       (kJ/mol)           -2.899898211733e+03
                       :           (eV)           -3.005532594413e+01
======================================================================


======================================================================
                            Polarizability
----------------------------------------------------------------------
 Frequency             :           (au)                      0.000000
                       :         (cm-1)                      0.000000
----------------------------------------------------------------------
 Total tensor          :            N/A            N/A            N/A
                       :            N/A            N/A            N/A
                       :      -0.000000      -0.000000       7.222013
 Isotropic average     :           (au)                           N/A
======================================================================


                                                                      
                                                                      
**********************************************************************
***                                                                ***
***                         Exiting MRChem                         ***
***                                                                ***
***                    Wall time :  0h  0m 24s                     ***
***                                                                ***
**********************************************************************
                                                                      
                                                                      
